### DISCUSSION TITLE
Creating a new node after a horizontal scroll causes the new node to move to some distance

### DISCUSSION THREAD

**Discussion by mdshamoon** [nyaruka/floweditor#1164] [Ideas]
## Issue
Creating a new node after a horizontal scroll causes the new node to move to some distance

Sharing the recording for reference

https://user-images.githubusercontent.com/32592458/158587261-e0ae5e0c-76b7-4646-89d7-f5ff0dbd51c5.mp4

**Comment by ericnewcomer** [comment]
I'm not seeing this with the current version of the editor. Are you able to reproduce this on TextIt?

**Comment by mdshamoon** [comment]
It's not replicable on TextIt. I am able to replicate by running the editor on my local machine. 
Maybe it gets fixed when used inside TextIt. Some overflow issues I think while just using the editor. 
Thanks for checking.
